scripts/update-versions.sh

```diff
@@ -36,5 +36,5 @@ sed -i '' -e "s/<revision>.*<\/revision>/<revision>$1<\/revision>/" ./parent/pom
 echo Updating checkstyle resources versions to $1 with Maven...
 mvn -pl modules/checkstyle versions:set -DnewVersion=$1 -DgenerateBackupPoms=false -DoldVersion=* -DprocessDependencies=false
 
-echo Updating .NET, C++ and other resources versions to $1 with Maven...
-mvn validate -P update-versions -D new.ignite.version=$1
+echo Updating .NET, C++ and other resources versions with Maven...
+mvn initialize -P thin
```

Comment:
Could not find `thin` profile anywhere in project

Reply:
Fixed.